### ISSUE TITLE
"compile" symlink causes problems

### ISSUE THREAD

**Issue by plicease** [hunspell/hunspell#358]
The symlink

compile => /usr/share/automake-1.15/compile

is causing me headaches, particularly on windows where symlinks in tarballs may be ignored, cause failure or something else weird (and infrequently it may actually create a symlink).  It is included in the 1.3.4 release, but wasn't in the 1.3.3 release.  Example failure:

```
Downloading File: v1.3.4.tar.gz ... Done
Extracting Archive ... No such file in archive: 'usr/share/automake-1.15/compile' at d:/lang/perl/strawberry/x86/5.22.1/perl/vendor/lib/Archive/Extract.pm li
ne 897.
Could not find file '/usr/share/automake-1.15/compile' in memory. at d:/lang/perl/strawberry/x86/5.22.1/perl/vendor/lib/Archive/Extract.pm line 897.
Making symbolic link 'D:\home\ollisg\dev\Alien-Hunspell\Alien-Hunspell-0.06\_alien\hunspell-1.3.4\compile' to '/usr/share/automake-1.15/compile' failed at d:
/lang/perl/strawberry/x86/5.22.1/perl/vendor/lib/Archive/Extract.pm line 897.
Unable to read 'D:\home\ollisg\dev\Alien-Hunspell\Alien-Hunspell-0.06\_alien\hunspell-1.3.4.tar.gz': Making symbolic link 'D:\home\ollisg\dev\Alien-Hunspell\
Alien-Hunspell-0.06\_alien\hunspell-1.3.4\compile' to '/usr/share/automake-1.15/compile' failed at D:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5/Alien
/Base/ModuleBuild.pm line 330.
No '/bin/tar' program found at D:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5/Alien/Base/ModuleBuild.pm line 330.
Extract failed due to errors at D:\home\ollisg\perl5\strawberry\x86\5.22.1\lib\perl5/Alien/Base/ModuleBuild.pm line 330.
Archive extraction failed! at Build line 61.
```

I can only assume that its inclusion was accidental, and has something to do with autotools, but I am inexpert in that to say exactly.

**Comment by plicease** [comment]
This demonstrates probably the easiest way to reproduce:

```
D:\home\ollisg\test\symlinks>perl -MArchive::Tar -E "my $tar = Archive::Tar->new; $tar->read('hunspell-1.3.4.tar.gz'); $tar->extract;"
No such file in archive: 'usr/share/automake-1.15/compile' at -e line 1.
Could not find file '/usr/share/automake-1.15/compile' in memory. at -e line 1.
Making symbolic link 'D:\home\ollisg\test\symlinks\hunspell-1.3.4\compile' to '/usr/share/automake-1.15/compile' failed at -e line 1.
Could not extract 'hunspell-1.3.4/compile' at -e line 1.
```

**Comment by reneengelhard** [comment]
yeah, https://lintian.debian.org/tags/package-contains-unsafe-symlink.html is emitted by Debians lintian tool for that, too

**Comment by dimztimz** [comment]
Building 1.3 and 1.4 were problematic for windows in various ways, not only this. The master is fixed, `compile` now is a shell script as I see. Use `git clone` instead downloading the tar.

Building the master on windows is can be done with any of the listed:

1. Visual Studio
2. MSYS2 + Mingw-w64 gcc

The old MSYS + Mingw are untested, but I think they should work too.

**Comment by plicease** [comment]
@dimztimz there is a link to the tar/zip files from the homepage (here: https://hunspell.github.io/ "Releases").  If the intent really is not to offer any tarballs (!!) then please at least update the homepage not to link to them.  Not having tarballs for production releases of hunspell will complicate deployment for us, I am sure others as well.

**Comment by dimztimz** [comment]
Tar balls are automatically provided by github and  they will continue to be provided.

What I am saying is that the bug is fixed in 1.5 which is still not released. And doing a quick fix just for 1.3.4 won't be done because I think that version has other problems when building on windows.

**Comment by plicease** [comment]
I ready "Use git clone instead of downloading the tar" to mean the tarballs weren't supported.  I can wait for 1.5.  Thanks for clarifying and for the fix!
